src/compiler/core.ts

```diff
@@ -2,6 +2,17 @@
 /// <reference path="performance.ts" />
 
 
+namespace ts {
+    export function startsWith(str: string, prefix: string): boolean {

```

Comment:
Aye. It was a useful tool which I was using in my tests. I could just copy
it, I suppose, if having them public is an issue.

On Thu, Aug 11, 2016, 2:40 PM Yui <notifications@github.com> wrote:

> In src/compiler/core.ts
> <https://github.com/Microsoft/TypeScript/pull/10159#discussion_r74508706>:
>
> > @@ -2,6 +2,17 @@
> >  /// <reference path="performance.ts" />
> >
> >
> > +namespace ts {
> > +    export function startsWith(str: string, prefix: string): boolean {
>
> question: is this intentional to not mark "internal"?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Microsoft/TypeScript/pull/10159/files/5ad76bd5fdd630ddc580ac497fc98d44f9acd4d1#r74508706>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACzAMigcs3CcrH9FBPolH9QTfGOZ7QNbks5qe5bPgaJpZM4JdPPR>
> .
>